test/journeys/specs/oneOnOne/dataApi/messaging.js

```diff
@@ -1,7 +1,17 @@
+import {assert} from 'chai';
+
 import testUsers from '@ciscospark/test-helper-test-users';
 import '@ciscospark/internal-plugin-conversation';
 
-import {sendMessage, verifyMessageReceipt, messageTests} from '../../../lib/test-helpers/space-widget/messaging';
+import {
+  canDeleteMessage,
+  deleteMessage,
+  flagMessage,
+  messageTests,
+  removeFlagMessage,
+  sendMessage,
+  verifyMessageReceipt
+} from '../../../lib/test-helpers/space-widget/messaging';
```

Comment:
maybe consider exporting another object containing all the helper functions (like how we did for `messageTests`) called `messageHelpers` or something similar?

Reply:
Yea, we need to decide on one way or the other. The helpers file is very mixed right now.